src/gluonts/model/canonical/_network.py

```diff
@@ -15,11 +15,11 @@
 from mxnet.gluon import HybridBlock
 
 from gluonts.core.component import validated
-from gluonts.model.common import Tensor
 
 # First-party imports
 from gluonts.mx.block.feature import FeatureEmbedder
 from gluonts.mx.block.scaler import MeanScaler
+from gluonts.mx.common import Tensor
```

Comment:
Not sure I like this `common` thing. (Same with dataset)

Couldn't this be `from gluonts.mx import Tensor`?

Reply:
sounds good 👍